src/main/kotlin/com/procurify/flagcounter/launchdarkly/LaunchDarklyResponses.kt

```diff
@@ -0,0 +1,72 @@
+package com.procurify.flagcounter.launchdarkly
+
+/**
+ * Representation of the https://apidocs.launchdarkly.com/reference#list-feature-flags response
+ */
+data class FlagResponse(
+        val totalCount: Int,
+        val items: List<FlagDetail>
+)
+
+/**
+ * Details of one feature flag
+ */
+data class FlagDetail(
+        val key: String,
+        val _maintainer: FlagMaintainer,
```

Comment:
Mostly just curious about this. Is the underscore prefix here is representing some version of a "private" property, or something like that? I was just curious around the reasoning for which of these properties are or aren't underscore-prefixed, because I don't really see anything conceptually separating them for me, and they seem to be accessed the same way throughout the code.

Reply:
It is the convention of the LaunchDarkly API. I shouldn't have it pollute the Kotlin names though. I think there is something like an `@SerializedAs("_maintainer")` that I can add, I will check